scripts/general-redirect.coffee

```diff
@@ -81,7 +81,8 @@ experts = [
   "tkuhrt",
   "xcr",
   "yacovm",
-  "zac"
+  "zac",
+  "kdenhartog"
```

Comment:
We have been attempting to alphabetize the names.

Reply:
thanks for pointing that out. I changed this now.